hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -121,7 +123,9 @@ public DeleteBlocksCommandHandler(OzoneContainer container,
         dnConf.getBlockDeleteThreads(), threadFactory);
     this.deleteCommandQueues =
         new LinkedBlockingQueue<>(dnConf.getBlockDeleteQueueLimit());
-    handlerThread = new Daemon(new DeleteCmdWorker());
+    long interval = this.conf.getLong(BLOCK_DELETE_COMMAND_WORKER_INTERVAL,
+        BLOCK_DELETE_COMMAND_WORKER_INTERVAL_DEFAULT);
```

Comment:
Get value from `dnConf`:

```suggestion
    long interval = dnConf.getBlockDeleteCommandWorkerInterval();
```

(Don't forget to remove unused imports.)